.travis.yml

```diff
@@ -9,20 +9,40 @@ env:
     - MINIKUBE_HOME=$HOME
     - MINIKUBE_VER=v0.28.2
     - K8S_VER=1.11.2
-    - KUBEVIRT_VER=v0.8.0-alpha.1
+    - KUBEVIRT_VER=v0.8.0
 
-matrix:
+jobs:
   include:
-  - name: Check syntax of all templates
-    env: TARGET=syntax-tests
-  - name: Functional test of the Fedora template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=fedora28
-  - name: Functional test of the Ubuntu template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=ubuntu1804
-#  - name: Functional test of the OpenSUSE template
-#    env: TARGET=functional-tests TEST_FUNCTIONAL=opensuse15
-  - name: Functional test of the CentOS/RHEL template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=rhel75
+    - stage: "Tests"
+      name: Check syntax of all templates
+      env: TARGET=syntax-tests
+      # -e is important to let make use the vars defined in "matrix:" above
+      script: make -e $TARGET
+    - script: make -e $TARGET
+      name: Functional test of the Fedora template
+      env: TARGET=functional-tests TEST_FUNCTIONAL=fedora28
+    - script: make -e $TARGET
+      name: Functional test of the Ubuntu template
+      env: TARGET=functional-tests TEST_FUNCTIONAL=ubuntu1804
+    - script: make -e $TARGET
+      name: Functional test of the CentOS/RHEL template
+      env: TARGET=functional-tests TEST_FUNCTIONAL=rhel75
+    - stage: "Deploy"
+      name: deploy 
+      before_script: skip
+      script: skip
+      before_deploy:
+      - cat presets/*.yaml templates/*.yaml > ./all.yaml
+      - tar -cvf release.tar all.yaml
+      deploy:
+        provider: releases
+        api_key: $GITHUB_OAUTH_TOKEN
+        file: "release.tar"
```

Comment:
Let's publish the plain yaml. That is much easier to fetch using curl.

For tarballs I'd recommedn to link to the provided tarball, i.e.: https://github.com/fabiand/common-templates/archive/v0.1.0.tar.gz

Reply:
That will link to the source though and that will not be good enough once we start compiling the templates.